iron-ajax.html

```diff
@@ -352,10 +359,10 @@
     /**
      * Performs an AJAX request to the specified URL.
      *
-     * @return {iron-request}
+     * @return {!IronRequestElement}
      */
     generateRequest: function() {
-      var request = document.createElement('iron-request');
+      var request = /** @type {!IronRequestElement} */ (document.createElement('iron-request'));
```

Comment:
Because this function is annotated as returning an IronRequestElement, we'll need to cast the element to the proper type. document.createElement simply returns an "Element". The cool thing about this is that everywhere we use request in this function is type checked now. If we want to, we can type the lastRequest property to ?IronRequestElement as well and then the compiler would verify the all uses of it and calls to _setLastRequest are valid.